Makefile

```diff
@@ -23,7 +23,7 @@ PHP_APACHE_URL ?= http://phpapacheapp
 RAILS_URL ?= http://railsapp:8020
 RUM_URL ?= http://rum:8000
 
-ES_USER ?= elastic
+ES_USER ?= admin
```

Comment:
Why does this need to change (did the `elastic` user not have any effect so far)?

Reply:
AFAICT, there is no `elastic` user and CI has only been passing because security has been disabled. We need security now (for Fleet), and when turning that on I noticed the tests failed because they couldn't auth with elastic:changeme.

The container has an `admin` user seeded: https://github.com/elastic/apm-integration-testing/blob/master/docker/elasticsearch/users